src/components/Api.js

```diff
@@ -23,3 +23,39 @@ export async function getFolderData() {
 
   return result;
 }
+
+export async function getUsersData(userId) {
+  const response = await fetch(`${BASE_URL}/users/${userId}`, {
+    method: 'GET',
+    headers: { accept: `*/*` },
+  });
+  if (!response.ok) throw new Error(`유저들데이터 ${BASE_ERROR_MESSAGE}`);
+  const result = await response.json();
+
+  return result;
```

Comment:
아래와 같이 간단하게 작성할 수 있답니다~! 

```suggestion
  return response.json();
```